chore: create security policy (#1322)
verdaccio/verdaccio: SECURITY.md

# Security Policy

## Supported Versions

Use this section to tell people about which versions of your project are
currently being supported with security updates.

| Version | Supported          |
| ------- | ------------------ |
| 2.x   | :x:                |
| 3.x   | :white_check_mark: |
| 4.x   | :white_check_mark: |

## Reporting a Vulnerability

At Verdaccio, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our team:

> * E-mail your findings to `verdaccio@pm.me`. If the report contains highly sensitive information, please consider encrypting your findings using our {PGP key}.

Author: @DanielRuf do you know what exactly we should define here `{PGP key}` ?

Reviewer: A link to the ASCII encoded public ksy (.asc file) probably.

Reviewer: A link to the ASCII encoded public key (.asc file) normally.

# Security Policy

## Supported Versions

Use this section to tell people about which versions of your project are
currently being supported with security updates.

| Version | Supported          |
| ------- | ------------------ |
| 2.x   | :x:                |
| 3.x   | :white_check_mark: |
| 4.x   | :white_check_mark: |

## Reporting a Vulnerability

At Verdaccio, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our team:

> * Report it either [Snyk Security Team](https://snyk.io/vulnerability-disclosure/) or [npmjs Security Team](https://www.npmjs.com/advisories/report?package=verdaccio), they will in contact with us in case of confirm the vulnerability.

Reviewer: they will in contact => they will be in contact

of confirm => of confirmation of
Or better: in the case that the vulnerability is confirmed.

Author: Actually, the point here is, they can contact us directly, but I'm sure their security teams have better tooling and experience testing the patch we can provide.

> * Report it either [Snyk Security Team](https://snyk.io/vulnerability-disclosure/) or [npmjs Security Team](https://www.npmjs.com/advisories/report?package=verdaccio), they will in contact with us in case of confirm the vulnerability.

Author: ```suggestion
* Report it either [Snyk Security Team](https://snyk.io/vulnerability-disclosure/) or [npmjs Security Team](https://www.npmjs.com/advisories/report?package=verdaccio), they will be in contact with us in case of confirming the vulnerability.
```

> * E-mail your findings to `verdaccio@pm.me`. If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).

Reviewer: We may be able to use <a href="mailto:..">`email`</a> or the markdown variant here.

> * E-mail your findings to `verdaccio@pm.me`. If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).

Author: ```suggestion
* E-mail your findings to sing [mailto](mailto:verdaccio@pm.me). If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).
```

Reviewer: To sing? Did you mean `[verdaccio@pm.me](mailto:verdaccio@pm.me)`?

> * E-mail your findings to sing [verdaccio@pm.me](mailto:verdaccio@pm.me). If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).

Reviewer: to sing [verdaccio@pm.me](mailto:verdaccio@pm.me) => to [verdaccio@pm.me](mailto:verdaccio@pm.me)

Rest is perfect =)

> * E-mail your findings to sing [verdaccio@pm.me](mailto:verdaccio@pm.me). If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).

Author: ```suggestion
* E-mail your findings to [verdaccio@pm.me](mailto:verdaccio@pm.me). If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).
```

Author: true :-)

Please follow these rules when testing/reporting vulnerabilities:
* Do not take advantage of the vulnerability you have discovered, for example by downloading more data than is necessary to demonstrate the vulnerability.

> * Do not read, modify or delete data that isn't you own.

Reviewer: you own => your own

> * Do not read, modify or delete data that isn't you own.

Author: ```suggestion
* Do not read, modify or delete data that isn't your own.
```

> * We ask that you do not to disclosure the problem to third parties until it has been resolved.

Reviewer: do not to disclosure => do not disclose
the problem => the findings (or keep it as is, we refer to "problem" in the rest of the text)

> * We ask that you do not to disclosure the problem to third parties until it has been resolved.

Author: ```suggestion
* We ask that you do not disclose the findings to third parties until it has been resolved.
```
